src/libs/actions/User.ts

```diff
@@ -181,10 +180,9 @@ function updateNewsletterSubscription(isSubscribed) {
 /**
  * Delete a specific contact method
  *
- * @param {String} contactMethod - the contact method being deleted
- * @param {Array} loginList
+ * @param contactMethod - the contact method being deleted
```

Comment:
It's weird only commenting one of the params. Perhaps commenting both in this case?

Reply:
I would just delete both `@param` notations here, I don't think they're providing any extra clarity